ui/src/childinfo/ChildInfo.js

```diff
@@ -62,25 +59,8 @@ export default class ChildInfo extends Component {
       });
     } else {
 
-      const { name, age, homeTown, illness, details } = this.state
-      const { type } = this.props
-
-      const wish = {
-        child: {
-          name: name,
-          age: age,
-          hometown: homeTown,
-          illness: illness
-        },
-        type: type ? type : '',
-        details: details
-      }
-
-      const response = await WishDetailsService.makeAWish(wish)
-
       this.setState({
-        showConfirmation: true,
-        childId: response._id
```

Comment:
Think we need this response._id saved, same as above

Reply:
added it back